design-toolkit/components/src/components/chip/types.ts

```diff
@@ -33,7 +33,7 @@ export type ChipListProps<T> = Omit<TagGroupProps, 'children'> &
   BaseChipProps;
 
 export type ChipProps = VariantProps<typeof ChipStyles> &
-  Omit<ComponentPropsWithRef<'div'>, 'size'> &
+  Omit<ComponentPropsWithRef<'div'>, 'size' | 'onClick'> &
```

Comment:
Sanity check this is intended

Reply:
And, this is because we want the `OnClick` from the `BaseChipProps`, correct?

Reply:
I think this just came along when I merged in `main` to this branch so I don't know the full context of the change

Reply:
This is ok. It came in with some the dep & ts updates. The props belong to a non-interactive element, and it's just a type deconfliction issue